src/index.css

```diff
@@ -889,12 +897,8 @@ table.table.corpus th:focus {
   height: 1.3rem;
 }
 
-.api-info {
-  white-space: nowrap;
-}
-
-.version-pill {
-  white-space: nowrap;
+.api-info, .version-pill {
+  white-space: pre-wrap;
```

Comment:
@markschwindt Why would we use `pre-wrap` here? The idea of `nowrap` was to avoid text wrapping so that we don't need to insert non breaking spaces. AFAIK, `pre-wrap` would not prevent.

Reply:
@cmil You are right, we don't. Tried to preserve text wrapping around "the pill" on smaller screen sizes, but there is a more elegant way to do that. Will link a new PR right away.